ci/lava/rb3gen2-vision-kit-boot.yaml

```diff
@@ -0,0 +1,71 @@
+actions:
+- deploy:
+    images:
+      image:
+        headers:
+          Authentication: Q_GITHUB_TOKEN
+        url: "{{BUILD_DOWNLOAD_URL}}"
+    postprocess:
+      docker:
+        image: ghcr.io/foundriesio/lava-lmp-sign:main
+        steps:
+        - export IMAGE_PATH=$PWD
+        - cp overlay*.tar.gz overlay.tar.gz
+        - echo "OVERLAY=overlay.tar.gz" >> $IMAGE_PATH/flash.settings
+        - echo "OVERLAY_PATH=/home/" >> $IMAGE_PATH/flash.settings
+        - echo "ROOTFS_IMAGE=disk-ufs.img2" >> $IMAGE_PATH/flash.settings
+        - echo "DEVICE_TYPE=debian-rb3gen2-vision-kit" >> $IMAGE_PATH/flash.settings
+        - cat $IMAGE_PATH/flash.settings
+    timeout:
+      minutes: 5
+    to: downloads
+- deploy:
+    images:
+      image:
+        url: downloads://{{BUILD_FILE_NAME}}
+      settings:
+        url: downloads://flash.settings
+      overlay:
+        url: downloads://overlay.tar.gz
+    timeout:
+      minutes: 5
+    to: flasher
+- boot:
+    auto_login:
+      login_prompt: 'login:'
+      username: debian
+      password_prompt: 'Password:'
+      password: debian
+      login_commands:
+      - sudo su
+    method: minimal
+    prompts:
+    - root@debian
+    - debian@debian
+    timeout:
+      minutes: 3
+- test:
+    definitions:
+    - from: git
+      name: "smoke-test"
+      path: automated/linux/smoke/smoke.yaml
+      repository: https://github.com/linaro/test-definitions.git
+      parameters:
+        SKIP_INSTALL: "True"
+        TESTS: "pwd, uname -a, ip a"
+- command:
+    name: network_turn_on
+context:
+  lava_test_results_dir: /home/lava-%s
+  test_character_delay: 10
+device_type: qcs6490
```

Comment:
I guess this is where we select target devices in the lab; I believe we have one vision kit and multiple core kits; could we specifically point at the vision kit? right now, vision kit images should boot on core kit and vice-versa, but we will want to specialize this progressively, notably as the DT for vision kit lands and we can start testing one of the two camera sensors with the vision kit